PorticoExportDom.inc.php

```diff
@@ -183,6 +185,73 @@ private function _buildArticle() : DOMElement {
 		return $root;
 	}
 
+	/**
+	 * Retrieve the file information from a galley
+	 * @param ArticleGalley $galley Galley instance
+	 * @return array
+	 */
+	private function _getFileInformation(ArticleGalley $galley) {
+		if (!($fileId = $galley->getData('submissionFileId'))) {
+			return null;
+		}
+		$fileService = Services::get('file');
+		$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
+		$submissionFile = $submissionFileDao->getById($fileId);
+		$file = $fileService->get($submissionFile->getData('fileId'));
```

Comment:
The `$fileService->getPath($fileId)` doesn't exist in my OJS 3.3, perhaps it was modified recently?!

Reply:
Yes -- see https://github.com/pkp/pkp-lib/commit/3945761b44436b021189ee90c8d58749ab859c85#diff-2a62568dd9cfa1070771bf6fe229ec11981532ebfe904cf2025fd378902c44dfL61.